client/container.go

```diff
@@ -656,6 +656,11 @@ func (c *Client) ContainerSetEACL(ctx context.Context, prm PrmContainerSetEACL)
 		panic("eACL table not set")
 	}
 
+	_, isCIDSet := prm.table.CID()
+	if !isCIDSet {
+		return nil, errors.New("missing container in eACL table")
```

Comment:
It feels strange writing this, but we should panic here. I'm not exactly excited about the way parameters are handled, but all methods like this one explicitly specify "Immediately panics if parameters are set incorrectly" and they behave according to this documentation. So we better stick to this pattern for now at least to preserve consistency.

Reply:
#390.

Reply:
replaced with panic